src/toast/ops/simple_jumpcorrect.py

```diff
@@ -179,27 +165,30 @@ def _find_peaks(self, toi, flag, flag_out, lim=3.0, tol=1e4, sigma_in=None):
             npeak = np.ma.sum(np.abs(mytoi) > sigma * lim)
 
         # Only one jump per iteration
-        if npeak > 0:
+        while npeak > 0:
             imax = np.argmax(np.abs(mytoi))
             amplitude = mytoi[imax]
             significance = np.abs(amplitude) / sigma
 
-            # Mask the peak for taking mean and finding additional peaks
-            istart = max(0, imax - tol)
-            istop = min(len(mytoi), imax + tol)
             # mask out the vicinity not to have false detections near the peak
+            istart = max(0, imax - tol)
+            istop = min(nsample, imax + tol)
             mytoi[istart:istop] = np.ma.masked
```

Comment:
Here the samples near the jump are masked for each subsequent iteration.  So this will find all jumps that are spaced at least 2 x `tol` apart.  After correcting those jumps, do we need to go back and look for other jumps that were in these masked regions?  In other words, a loop over `_find_peaks()` and `_remove_jumps()`?

Reply:
`tol` is half of `filterlen` [1] [2]. If there are two jumps within one filter length, it probably won't even find correct jump location. The peak location may not be the jump location. I'd say this is the limitation of this method.

[1] https://github.com/Bai-Chiang/toast/blob/ceb8d47e3c56a3ea4032d358242e29893b941340/src/toast/ops/simple_jumpcorrect.py#L268
[2] https://github.com/Bai-Chiang/toast/blob/ceb8d47e3c56a3ea4032d358242e29893b941340/src/toast/ops/simple_jumpcorrect.py#L126

Reply:
Good point.  Sounds like this is ready to merge then.  Thanks!